app/src/main/java/io/left/meshim/controllers/RightMeshController.java

```diff
@@ -63,9 +68,9 @@
     private IActivity callback = null;
     //reference to service
     private MeshIMService meshIMService;
-    // keeps track of all the undeliveredMessages.
+    // keeps track of all the undeliveredPackages.
     private HashMap<Integer, Integer> unDeliveredMessageIds = new HashMap<Integer, Integer>();
-
+    private ConcurrentMap<Integer,MeshId> undeliveredPeerChangeMessage = new ConcurrentHashMap<>();
```

Comment:
It's a "peer change" in RightMesh, but the meshIM message is called a "Peer Update."